app/src/main/java/de/tum/in/tumcampusapp/component/ui/onboarding/StartupActivity.kt

```diff
@@ -180,6 +180,11 @@ class StartupActivity : BaseActivity(R.layout.activity_startup) {
 
     private fun openMainActivity() {
         val intent = Intent(this, BaseNavigationActivity::class.java)
+        // If the app has been restarted after a language change,
+        // the intent includes an extra which is passed on to BaseNavigationActivity to open up the settings again.
+        if (getIntent().getBooleanExtra("SETTINGS", false)) {
```

Comment:
I think there should be some kind of Const class. Can you merge all the "SETTINGS" there, so its used consistently?

Reply:
I do not understand what you mean. Consts are constant. I use this extra to return to the settings activity if the app has been restarted after a language change. I am not sure how a Const class could help me here. Maybe you could explain what you mean or submit a commit yourself ;)

Maybe the name of the extra is just a little confusing. It is not actually a setting but just a kind of information which is only needed after the restart of the app.

Reply:
Sorry, I see how this was too ambiguous. 

In the `utils` package you will find a `Const.kt` class. There we keep track of strings that are used throughout the app. This helps with consistency and also avoids having naming conflicts. By giving the string a variable name, the code also becomes more readable.

Reply:
Ah, thanks. It's done.